packages/neo4j-driver/test/index.test.js

```diff
@@ -139,7 +139,7 @@ describe('#unit index', () => {
       }
     })
 
-    describe('await rxRession.beginTransaction().toPromise() return', () => {
+    xdescribe('await rxRession.beginTransaction().toPromise() return', () => {
```

Comment:
is that related to the other PR with rx changes?

Reply:
ops, it is a leftover. I was skipping the test here for checking for another issues.